WalletWasabi.Fluent/ViewModels/TransactionBroadcasting/BroadcastTransactionViewModel.cs

```diff
@@ -33,7 +33,7 @@ public BroadcastTransactionViewModel(Network network, SmartTransaction transacti
 
 	public string? InputAmountString { get; set; }
 
-	public string? FeeString { get; set; }
+	public string FeeString { get; set; } = "Unknown";
```

Comment:
Setting a value here is not needed anymore.

Reply:
To fix `null` warning as it's not nullable now.